### DISCUSSION TITLE
Stop whisper from removing "Repairs"

### DISCUSSION THREAD

**Discussion by GiulioRosani** [openai/whisper#2081] [Q&A]
For a research project in the field of Humanities, we are currently using Whisper to transcribe multiple interviews. As part of the project is analyzing the occurrence and use of "repairs" (ehm, uhm, like, you know, or even stopping and restarting a sentence), we were wondering if it is possible to tell Whisper (from command line) to avoid fixing such "repairs". 
By looking at the discussion here and elsewhere, I've seen that you can start the transcription with a prompt, but the use of this feature seems not ideal in our case. I also checked the "whisper --help" command and potentially identified "temperature" as an interesting parameter, as well as "patience factor". But since I don't really understand what they do from the documentation I could find, I wanted to ask the community if they had some input in this matter.

**Comment by ryanheise** [comment]
You can try using `--initial_prompt ".............."` and engineer a prompt that includes Ums and Ahs in a way that is typical of the speech you are transcribing. It will influence Whisper to continue in the same style. Careful, though, if your prompt lacks punctuation, you might influence Whisper to not use punctuation. Since the Whisper tokenizer interprets the space as belonging to the beginning of a word, you might find it helpful to include a space before even the first word of the sentence. E.g. `" Um, so let's get started and, ah, see how we go."`
